app/modules/entities/components/contributions.svelte

```diff
@@ -0,0 +1,16 @@
+<script>
+  import ContributionsPeriod from './contributions_period.svelte'
+</script>
+<div id="activityLayout">
```

Comment:
Could we keep as a convention to add one empty line as padding between script/template/style blocks? That improves readability imo

Reply:
Also, `activityLayout` -> `contributonsLayout`?

Reply:
yeah not really, the more it goes the more empty lines only make files longer to scroll, i would actually be for deleting all empty lines of the project :)

Reply:
many empty lines are the sign that a function is too big and should be split in sub functions, and eventually in different module files (yeah modularization!), but in the case of Svelte components, we can't just split those 3 distinct roles over 3 files, they have to live in the same file, while serving different purposes, so separating by an empty line seems the decent thing to do (and [what people seem to be doing](https://github.com/search?l=Svelte&q=style&type=Code))

Reply:
those text blocks are distinctive enough to me, from js to html to css, within the first second i can see which is which, therefore an extra line to distinct within the same 'language' makes sense but not here, it only makes you scroll more